src/store/customer/reducer.js

```diff
@@ -0,0 +1,12 @@
+import { SET_CUSTOMER } from './actions'
+
+const reducer = (state = {}, action) => {
+  switch (action.type) {
+    case SET_CUSTOMER:
+      return action.payload
+    default:
+      return state
+  }
+}
+
+export default reducer
```

Comment:
https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html

Reply:
Very nice article, I faced the problem in another project already, but was not entirely sure if it is really bad practise or I miss something. This is clear now:)